setup.cfg

```diff
@@ -61,7 +61,6 @@ source =
 omit =
   tests/*
   setup.py
-  coalib/misc/Asyncio.py
```

Comment:
this needs to be listed in `omit` - this is why the PR build is failing - there is a moban variable which controls this list, and this file needs to be added to the list.

Reply:
Will add it to file in mobans repo